tests/e2e/vsphere.go

```diff
@@ -1069,3 +1069,15 @@ func (vs *vSphere) verifyLabelsAreUpdated(volumeID string, matchLabels map[strin
 	}
 	return nil
 }
+
+//verifyDatastoreMatch verify is dsurl matches with given one for the volumeid
+func (vs *vSphere) verifyDatastoreMatch(volumeID string, dsUrl string) bool {
+	queryResult, err := vs.queryCNSVolumeWithResult(volumeID)
+	framework.Logf("queryResult: %s", spew.Sdump(queryResult))
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
```

Comment:
Incase of error this queryResult would be Nil - better check for the error not occured at first place and then print the query result

Reply:
done